__tests__/V4KVMClusterManagement.js

```diff
@@ -91,12 +91,20 @@ it('renders all the v4 KVM cluster data correctly without nodes ready', async ()
     expect(portsContainer).toContainHTML(html);
   });
 
-  const nodes = getByText('Nodes').nextSibling;
-  const nodesRunning = getNumberOfNodes({
-    ...v4KVMClusterResponse,
-    status: v4KVMClusterStatusResponse,
-  });
-  expect(nodes).toHaveTextContent(nodesRunning);
+  const nodes = getByText('Nodes').nextSibling.textContent;
+  expect(nodes).toBe('0');
+});
+
+/******************** PENDING TESTS ********************/
+
+it.skip('renders all the v4 KVM nodes', async () => {
```

Comment:
I guess there are some changes in the tests still to do?

Reply:
Test removed